app/sidekiq/evss/disability_compensation_form/submit_uploads.rb

```diff
@@ -3,15 +3,56 @@
 module EVSS
   module DisabilityCompensationForm
     class SubmitUploads < Job
-      STATSD_KEY_PREFIX = 'worker.evss.submit_form526_upload'
```

Comment:
double check this `*_PREFIX` isn't referenced from outside the class.

Reply:
Confirmed there are no references in the codebase to `STATSD_KEY_PREFIX` associated with any of the jobs where this constant was renamed `STATSD_KEY`.